structuralcodes/sections/_generic.py

```diff
@@ -196,7 +196,7 @@ def compute_area_moments(geometry, material=None):
                 iyy,
                 iyz,
                 tri,
-            ) = self.integrator.integrate_strain_response_on_geometry(
+            ) = self.integrator.integrate_strain_response_on_geometry_stress(
```

Comment:
Perhaps the name is starting to become a bit long? What if we simply call the method `stress_resultant`?